resources/replicaset.go

```diff
@@ -97,30 +83,17 @@ func (r ReplicaSet) Create() error {
 	return err
 }
 
-func (r ReplicaSet) Status() (string, error) {
-	return replicaSetStatus(r.Client, r.ReplicaSet.Name, r.Meta)
+func (r ReplicaSet) Status(meta map[string]string) (string, error) {
+	return replicaSetStatus(r.Client, r.ReplicaSet.Name, meta)
 }
 
 func NewReplicaSet(replicaSet *extensions.ReplicaSet, client unversioned.ReplicaSetInterface) ReplicaSet {
-	return ReplicaSet{ReplicaSet: replicaSet, Client: client, Meta: make(map[string]string)}
+	return ReplicaSet{ReplicaSet: replicaSet, Client: client}
 }
 
 type ExistingReplicaSet struct {
 	Name   string
 	Client unversioned.ReplicaSetInterface
-	Meta   map[string]string
-}
-
-func (r ExistingReplicaSet) UpdateMeta(meta map[string]string) error {
-	for key, value := range meta {
-		oldValue, exists := r.Meta[key]
-		if exists && oldValue != value {
-			return fmt.Errorf("Resource %v already has meta %v set to %v. Can not set it to %v",
-				r, key, oldValue, value)
-		}
-		r.Meta[key] = value
-	}
-	return nil
```

Comment:
This deserves for update in https://github.com/Mirantis/k8s-AppController/blob/0e9a8db8836d4b70322712a2f924dcb09f4d9646/README.md#dependencies